include/openssl/ssl.h

```diff
@@ -756,41 +756,41 @@ void SSL_CTX_set_next_protos_advertised_cb(SSL_CTX *s,
                                            int (*cb) (SSL *ssl,
                                                       const unsigned char
                                                       **out,
-                                                      unsigned int *outlen,
+                                                      size_t *outlen,
                                                       void *arg), void *arg);
 void SSL_CTX_set_next_proto_select_cb(SSL_CTX *s,
                                       int (*cb) (SSL *ssl,
                                                  unsigned char **out,
-                                                 unsigned char *outlen,
+                                                 size_t *outlen,
                                                  const unsigned char *in,
-                                                 unsigned int inlen,
+                                                 size_t inlen,
                                                  void *arg), void *arg);
 void SSL_get0_next_proto_negotiated(const SSL *s, const unsigned char **data,
                                     unsigned *len);
 # endif
 
-__owur int SSL_select_next_proto(unsigned char **out, unsigned char *outlen,
-                          const unsigned char *in, unsigned int inlen,
-                          const unsigned char *client,
-                          unsigned int client_len);
+__owur int SSL_select_next_proto(unsigned char **out, size_t *outlen,
```

Comment:
That is why I asked. I think it makes sense, since 1.1 is breaking ABI compatibility, to update with proper size types. But, yeah, it's not 100% source compatible.